explainer.md

```diff
@@ -72,6 +72,113 @@ There are still many open questions, but a few big ones:
+
+The API is designed to expect the use of response encryption so that this PII is exposed only to the requesting server and not any code running in the web page or browser. It is an [open question](https://github.com/WICG/digital-identities/issues/109) whether this response encryption is something we can reasonably enforce at this layer or not.
+
+The API is also designed to require request transparency to enable user agents and operating systems to appropriately inform users about the level of privacy risk involved in the request. A major area of outstanding work is to outline in the specification our privacy and security guidance to implementers, but we already know that the [Chromium implementation](https://docs.google.com/document/d/1L68tmNXCQXucsCV8eS8CBd_F9FZ6TNwKNOaFkA8RfwI/edit) intends to show users stronger warnings in riskier scenarios such as those that lack the use of selective disclosure.
+
+> 03.  Do the features in your specification expose personal information,
+>      personally-identifiable information (PII), or information derived from
+>      either?
+
+Yes, the API is designed to help facilitate communication of PII from secure wallet applications to verifier web servers. 
+
+> 04.  How do the features in your specification deal with sensitive information?
+
+Today, online identity verification (e.g., for KYC) is usually done by submitting photos of government identity documents (for example, uploading photos containing the PII via the web’s `<input type=file>` mechanism. Due to the privacy and security limits of this approach, credential issuers and verifiers are working to move to the use of wallet applications which can selectively disclose cryptographically-attested properties. Today, those approaches rely on generic web→app communication paths like custom schemes and server-to-server communication. 
+
+The use of wallets aims to improve on the status quo of uploading images in a number of ways:
+ * Enable selective disclosure (e.g., for more privacy-friendly age≥18 verification)
+ * Enable end-to-end PII encryption between the wallet application and the relevant verification server
+ * Enable request authentication, where a wallet is required to cryptographically verify that the requester has permission to access the credential
+
+Further, this specification aims to improve over the existing communication channels used by websites to talk to wallet apps by:
+ * Enabling browsers and operating systems to provide meaningful credential selection and permission screens to users, prior to wallet applications becoming aware of the presentation request
+ * Securely communicating the origin of the requesting site to the wallet application, so that it can implement its own MITM protection with support of WebPKI
```

Comment:
```suggestion
 * Securely communicating the origin of the requesting site to the wallet application, so that it can implement its own MITM (man-in-the-middle) protection with support of WebPKI
```